### DISCUSSION TITLE
for using tools/misc/clip_feature_extraction.py in mmcv==2.0.0

### DISCUSSION THREAD

**Discussion by flashult** [open-mmlab/mmaction2#2479] [Q&A] [answered]
i want to use 'tools/misc/clip_feature_extraction.py' in current version of mmaction2 because i want to use clip_feature_extraction.py with recent models.
it works well in branch 0.x dependency.
but in current dependency, clip_feature_extraction.py doesn't work because some function are removed in mmcv==2.0.0(ex. mmcv.parallel)
Any good ideas to solve this situation?

**Comment by cir7** [comment, marked as answer]
Sorry for the bug, the script is outdated, we will fix it asap.
